airbyte-integrations/base-singer/base_singer/singer_helpers.py

```diff
@@ -34,6 +34,36 @@ class Catalogs:
 
 
 class SingerHelper:
+    @staticmethod
```

Comment:
having multiple types is valid jsonschema. i don't think we want to normalize it out in the integration if we are saying that you define your stream's schema using jsonschema.

Reply:
oh. saw your comment here: https://github.com/airbytehq/airbyte/pull/609#pullrequestreview-511408903. maybe we're on the same page now.

Reply:
Yup, we are. I was confused. JSONSchema it is.

Reply:
In which case please merge your PR 😆